proposals/accepted/047-standard-library-reform.rst

```diff
@@ -0,0 +1,559 @@
+Standard library reform
+=======================
+
+.. sectnum::
+.. contents::
+
+Abstract
+--------
+
+Issues with the standard library are holding back the Haskell ecosystem.
+The problems and solutions are multifaceted, and so the Haskell Foundation in its "umbrella organization" capacity is uniquely suited to coordinate the fixing of them.
+
+The problems are briefly as follows:
+
+#. Major version bumps every compiler release is an inexcusable nuisance.
+
+#. No clear boundary between GHC private/unstable library support code and public/stable standard library interfaces.
+   `CLC Issue #015`_.
+
+#. No clear portability guarantees with new supported platforms like the web browser and Web Assembly System Interface (WASI).
+
+#. Breaking changes (when we do want them) have to be coupled with GHC versions and rather than staggered, which is painful.
+
+#. Popular and uncontroversial machinery like ``Text`` is not available from the standard library.
+
+By reshuffling our interfaces and implementations a like, we should be able to solve all these problems.
```

Comment:
```suggestion
By reshuffling our interfaces and implementations, we should be able to solve all these problems.
```

I think "a like" was supposed to be "alike", but if so I think the word is redundant. It seems a bit flowery or poetic. Doesn't "By reshuffling our interfaces and implementations" suffice?